scripts/check-update.sh

```diff
@@ -14,8 +14,8 @@ fi
 CHANNEL=$1
 echo "CHANNEL: $CHANNEL"
 
-if [ "$CHANNEL" = "dev" ]; then
-    CURRENT_VERSION=$(cat scripts/prisma_version_unstable)
+if [ "$CHANNEL" = "insider" ]; then
+    CURRENT_VERSION=$(cat scripts/prisma_version_insider)
```

Comment:
This probably means we need to make sure this is created with the correct content on merge of this PR?

Reply:
This should no be a problem I think, @divyenduz?

Reply:
The channel should be called "dev" here. 

We should add the value of the last Prisma CLI version for which an extension was created. 

Probably renaming the file `prisma_version_unstable` to `prisma_version_insiders`. 

We can also keep the internal naming as unstable/stable and just change the extension name to insiders. I don't see the need to change the rest of the code/internal details unless there is a strong reason.